composer.json

```diff
@@ -36,7 +36,10 @@
         "php-http/mock-client": "^0.3.2",
         "symfony/process": "^2.3||^3.0",
         "symfony/http-kernel": "^2.3||^3.0",
-        "phpunit/phpunit": "^5.7 || ^6.0"
+        "phpunit/phpunit": "^5.7 || ^6.0",
+        "toflar/psr6-symfony-http-cache-store": "dev-master",
+        "symfony/lock": "^3.4@dev",
+        "symfony/cache": "^3.4@dev"
```

Comment:
we need to adjust this once the psr6-symfony-http-cache-store and symfony 3.4 are released.